basic1/scripts/script.js

```diff
@@ -0,0 +1,25 @@
+const firstName = document.querySelector("#firstName");
+const lastName = document.querySelector("#lastName");
+const fullName = document.querySelector("#fullName");
+const re = new RegExp("^[A-Za-z]+$");
+/**
+ * This Function Updates the value of the disabled FullName input field.
+ */
+const update = function () {
```

Comment:
Please add more details about this function like it gets the string from firstname & lastname and concat it in fullname field.
Provide an empty line before starting multi line comments.